microbiome.niaiddata.org/manifest.json

```diff
@@ -4,21 +4,23 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "quay.io/cdis/arborist:master",
+    "arborist": "quay.io/cdis/arborist:1.2.1",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
-    "fence": "quay.io/cdis/fence:master",
+    "fence": "quay.io/cdis/fence:fix_niaddata",
```

Comment:
noooo 😆 
this branch doesn't exist anymore, it was probably was merged - can we pin pls?

Reply:
omg I just copied from aids... sure will fix that